recipe_scrapers/sallysbakingaddiction.py

```diff
@@ -29,7 +30,21 @@ def yields(self):
         return self.schema.yields()
 
     def image(self):
-        return self.schema.image()
+        image = self.data.get("image")
+
+        if image is None:
+            raise SchemaOrgException("Image not found in SchemaOrg")
+
+        if isinstance(image, list):
+            image = image[-1]
+
+        if isinstance(image, dict):
+            image = image.get("url")
+
+        if "http://" not in image and "https://" not in image:
```

Comment:
It would be nice to keep the comment that appeared below this line in the `SchemaOrg` original.  Note that that's likely to be updated soon though, by pull request #1032.

Reply:
Good catch, not sure why I removed that, think I caught myself in two minds between the initial version and more defensive version. If we go with this approach will add it back, better in your opinion to use the current one for consistency now and update after that PR gets merged, or just use the new version now?